examples/syscall.c

```diff
@@ -14,15 +14,30 @@
 #include <linux/kernel.h>
 #include <linux/module.h>
 #include <linux/moduleparam.h> /* which will have params */
-#include <linux/syscalls.h>
-#include <linux/unistd.h> /* The list of system calls */
+#include <linux/unistd.h>      /* The list of system calls */
+#include <linux/version.h>
 
 /* For the current (process) structure, we need this to know who the
  * current user is.
  */
 #include <linux/sched.h>
 #include <linux/uaccess.h>
 
+/* The ksys_close in function aquire_sys_call_table is no longer support
+ * after kernel 5.11, we need to replace it with close_fd instead.
+ */
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 11, 0))
+#include <linux/syscalls.h> /* For ksys_close */
+#else
+#include <linux/fdtable.h> /* For close_fd */
+#endif
+
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 11, 0))
+#define CLOSE_FUNC ksys_close
```

Comment:
You can simply define `close_fd` as `ksys_close`. That is,
```c
#define close_fd ksys_close
```

And move the definition right after the inclusion `#include <linux/syscalls.h>`.